Sources/SwifterSwift/MapKit/MKPolylineExtensions.swift

```diff
@@ -21,6 +21,16 @@ public extension MKPolyline {
         self.init(coordinates: &refCoordinates, count: refCoordinates.count)
     }
 
+    /// SwifterSwift: Create a new MKPolyline from a provided Array of locations.
+    ///
+    /// - Parameter locations: Array of CLLocation(s).
+    convenience init(locations: [CLLocation]) {
```

Comment:
Maybe this would be nicer if we take an `[CLLocationCoordinate2D]` instead?

Reply:
There already exists one with that arg. I just thought it would be nice for users to have cllocation also.

Reply:
Humm, I see now :)